### PR TITLE
reverted a commit, couple other small fixes

### PR DESCRIPTION
**Pull request by cblanto7**
Reverted the push that was made last night, and saw that building needed a getter/setter in the Room class or else it came up null in thyme leaf. made an attempt at a section http integration test but did not get far

### CODE REVIEW

File: src/main/resources/templates/section.html [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
 <div layout:fragment="content">
-    <h1>Section <span th:text="${section.number}"></span></h1>
+    <h1>Section <span th:text="${section.number + ' ' + section.course.getName()}"></span></h1>
     <div class="container">
         <h6>Room:</h6>
         <div class="container">
             <table class="table">
                 <tr >
-                    <td th:text="${section.room?.name + ' ' + section.room?.building?.name}"></td>
+                    <td th:text="${section.room?.name + ' ' + section.room?.getBuilding().getName()}"></td>
```
Comment (Brodski) [inline src/main/resources/templates/section.html]: I think this change is vulnerable to getting a null value, and thus vulnerable to crashing the html. BUT if you assure me that "it actually is safe" then I will gladly accept it

Comment (cblanto7) [reply src/main/resources/templates/section.html]: ok I will add the `?`

**Comment by Brodski** [comment]
I must have missed this previously, but sectionController is using the old update instead of the new one. That was also something I wanted to fix yesterday as a PR, but accidentally merged it in

**Comment by cblanto7** [comment]
sectionController is using a new version of update that I have been working on for the past few days, sample it if you want but I have tested it throughly and it works well.

File: src/main/resources/templates/course.html [hunk 2]
```diff
@@ -46,7 +46,7 @@ <h5>Sections</h5>
                                     <!--<div class="col">Building: <b th:text="${section.room?.building?.name}"/></div>-->
                                     <div class="col">Schedule: <b th:text="${section.schedule}"/></div>
                                     <div class="col">Time: <b th:text="${section.getTimeLength()}"/></div>
-                                    <div class="col">Semester: <b th:text="${section.semester?.year + ' ' + section.semester?.season}"/></div>
+                                    <div class="col">Semester: <b th:text="${section.semester?.year + ' ' + section.semester.season}"/></div>
```
[the remark this answers is not in the document]
Comment (cblanto7) [reply src/main/resources/templates/course.html]: ok changing this now

**Pull request merged**